timed/reports/views.py

```diff
@@ -209,6 +209,7 @@ def _create_workreport(self, from_date, to_date, today, project, reports,
         tmpl = settings.WORK_REPORT_PATH
         doc = opendoc(tmpl)
         table = doc.sheets[0]
+        tasks = {}
```

Comment:
you could use defaultdict to avoid below if/else.

https://docs.python.org/3/library/collections.html#collections.defaultdict

Reply:
Done